migration/planning-migration-3-to-4.adoc

```diff
@@ -0,0 +1,194 @@
+[id="planning-migration-3-to-4"]
+= Planning your migration
+include::modules/common-attributes.adoc[]
+:context: planning-migration-3-to-4
+
+toc::[]
+
+Before performing your migration to {product-title} 4.2, it is important to take the time to properly plan for the migration. {product-title} 4 introduces architectural changes and enhancements, so the procedures that you used to manage your {product-title} 3 cluster might not apply for {product-title} 4.
+
+This document provides high-level information on the most important xref:../migration/planning-migration-3-to-4.adoc#migration-comparing-ocp-3-4[differences between {product-title} 3 and {product-title} 4] and the most noteworthy xref:../migration/planning-migration-3-to-4.adoc#migration-considerations[migration considerations]. For detailed information on configuring your {product-title} 4 cluster, review the appropriate sections of the {product-title} documentation. For detailed information on new features and other notable technical changes, review the xref:../release_notes/ocp-4-2-release-notes.adoc#ocp-4-2-release-notes[OpenShift Container Platform 4.2 release notes].
+
+Note that it is not possible to upgrade your existing {product-title} 3 cluster to {product-title} 4. You must start with a new {product-title} 4 installation. Tools are available to assist in migrating your control plane settings and application workloads.
+
+[NOTE]
+====
```

Comment:
I might move this information out of the NOTE box and to earlier in the introduction, perhaps between the first and second paragraph. Just a suggestion.